### PR TITLE
[Release] Moya 11.0.0-beta.1

### PR DESCRIPTION
**Pull request by SD10**
This PR cuts the Moya 11.0.0-beta.1 release from the `development` branch.

We currently have some work from `master` that is not present in `development`. My solution to that was to merge `master` into `development` with this PR. I'm not sure if this is usually done differently and I know we have a different workflow this release.

cc @sunshinejr

### CODE REVIEW

File: Sources/Moya/TargetType.swift [hunk 1]
```diff
@@ -18,8 +18,13 @@ public protocol TargetType {
     /// The type of HTTP task to be performed.
     var task: Task { get }
 
+<<<<<<< HEAD
```
Comment (SD10) [inline Sources/Moya/TargetType.swift]: this needs to be removed :joy:

**Comment by SD10** [comment]
It looks like this could the integration tests failing from #1424?

**Comment by sunshinejr** [comment]
@SD10 yeah it seems like this is the case with integration tests taking longer than expected. 

Also, I'm gonna take a look at this PR ASAP, sorry for delay.

File: Changelog.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # Next
+
+# [11.0.0-beta.1] - 2017-1-9
```
Comment (sunshinejr) [inline Changelog.md]: AFAIK this should be done by rake releasing script

File: Moya.podspec [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "Moya"
-  s.version      = "10.0.1"
+  s.version      = "11.0.0-beta.1"
```
Comment (sunshinejr) [inline Moya.podspec]: This should be done by the script as well

**Comment by SD10** [comment]
@sunshinejr Ok I'm going to try and do it that way

**Comment by BasThomas** [comment]
Thanks for taking this on @SD10! 😀

**Pull request closed without merging**